src/components/ProgressBar/index.tsx

```diff
@@ -10,31 +10,40 @@ export interface ProgressBarProps {
   height?: number;
 }
 
-export const ProgressBar = ({ value = 0, height = 4, backgroundColor, barColor, ...restProps }: ProgressBarProps) => (
-  <Container backgroundColor={backgroundColor} height={height} {...restProps}>
-    <Bar barColor={barColor} value={Number(value) > 100 ? 100 : value} />
-  </Container>
+export const ProgressBar = memo(
+  ({ value = 0, height = 4, backgroundColor = gray200, barColor = orange600, ...restProps }: ProgressBarProps) => {
+    const barValue = Number(value);
+
+    let transform = barValue > 100 ? 100 : barValue;
+    if (barValue < 0) {
+      transform = 0;
+    }
```

Comment:
```suggestion
    const percent = Math.max(0, Math.min(100, barValue));
```

이 값은 특정 css와 엮여있다기보단 percent 값을 뜻하니, 요런 이름이 어떨까요?

(+ 민맥스를 이용한 number range 처리)